docs/src/css/custom.css

```diff
@@ -25,6 +25,7 @@
   --slack-dark-blue: #1264a3;
 
   --grey: #868686;
+  --dim: #eef2f6;
```

Comment:
nice nice -- a good reminder to myself i still need to set up a github workflow to match css! I'll probably come ask the github workflow expert for help in that at some point 👀